internal/reconcilers/entity_add.go

```diff
@@ -52,31 +52,25 @@ func (r *Reconciler) handleEntityAddEvent(msg *message.Message) error {
 		return nil
 	}
 
-	var repoOwner string
-	var repoName string
-	var ok bool
-	if repoOwner, ok = event.Entity["repoOwner"].(string); !ok {
-		return errors.New("invalid repo owner")
+	if event.EntityType != pb.Entity_ENTITY_REPOSITORIES {
```

Comment:
Does this really only work for repos?

Reply:
At the moment, those handlers go directly to the repository service, see https://github.com/stacklok/minder/blob/d78ff1c866655ef6cb22fb3b6267f2d3060b0ccc/internal/reconcilers/entity_delete.go#L63 or https://github.com/stacklok/minder/blob/d78ff1c866655ef6cb22fb3b6267f2d3060b0ccc/internal/reconcilers/entity_add.go#L91

The type parameter was really redundant as it was not checked anywhere. This change at least made the code fail predictably with another type, previously you'd send data and the code would still try to create a repo out of them..